chimg/chroot.py

```diff
@@ -687,3 +690,113 @@ def _mount(self):
             logger.info("mount points setup done")
             yield
         logger.info("mount points cleanup done")
+
+    def create_manifest(
```

Comment:
I don't think `chimg` should handle any manifest/SBOM creation by its own. This should be done by other tools .
The goal of `chimg` is, to have a declarative way to modify a chroot. That's it.

Reply:
use another tool in the post cmd . or another tool that can chroot into a directory. or antother tool that can create a sbom from a raw image. 
There are integration points (like post cmd) which can be used. I'm just saying that the code that generates the manifest should not be part of chimg.

Reply:
I'm now interested in this discussion, as we may want or need manifest generation capabilities for customers we direct to use this tool. We have cpc-sbom for sboms, but creating manifests for cpc is all in livecd-rootfs. I generally agree that having a manifest tool would be nice, but it's basically just one dpkg-query command. I wonder if it's worth putting _only_ that functionality here, and leaving sbom generation to the cpc-sbom tool? or else moving the dpkg-query (and filelist) functionality into cpc-sbom?